libs/langchain/langchain/callbacks/__init__.py

```diff
@@ -30,6 +31,10 @@
 
 
 def __getattr__(name: str) -> Any:
+    try:
+        return importlib.import_module("." + name, __name__)
```

Comment:
I think we need a lookup table here defining the things to export

Reply:
bc there might be things we don't want to export?

Reply:
this kind of use could be an attack vector. Better to lock down and only export the things we explicitly want exported.

e.g. someone could find a way to resolve to non-langchain libs, relative `..` imports, etc.

Reply:
^ +1 exactly what i was thinking